security/webauthn/src/test/java/AbstractWebAuthnTest.java

```diff
@@ -0,0 +1,110 @@
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.restassured.RestAssured;
+import io.restassured.filter.Filter;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpMethod;
+
+public abstract class AbstractWebAuthnTest {
+
+    protected abstract RestService getApp();
+
+    private static HttpClient httpClient = Vertx.vertx().createHttpClient();
```

Comment:
You are not closing neither Vertx or HttpClient, while this creates new instance of both. I think it should be closed or it can lead to resource leak. WDYT?

Reply:
Yes it should be closed it as it is commented on and suggested in the other PR opened that we have: #1408